charts/sdfactory/values.yaml

```diff
@@ -52,14 +52,14 @@ podSecurityContext:
   fsGroup: 2000
 
 securityContext:
-    # -- Controls whether a process can gain more privileges
+  capabilities:
+    drop:
+      - ALL
   allowPrivilegeEscalation: false
-  # capabilities:
-  #   drop:
-  #   - ALL
   # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  runAsUser: 1000
+  runAsNonRoot: true
+  runAsUser: 1001
+  runAsGroup: 3000
```

Comment:
could you please just explain with you user that your and group? and not [those](https://github.com/catenax-ng/tx-sd-factory/blob/context-changes/Dockerfile#L41)

Reply:
Sorry, I have corrected it as per Dockerfile